src/components/Layout/SymptomsCarousel/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import SymptomsCards from '../SymptomsCards'
+import './SymptomsCarousel.scss';
+import '../../../styles/_colors.scss';
+import Tiredness from '../../../assets/images/tiredness.svg';
+import Cough from '../../../assets/images/cough.svg';
+import Fever from '../../../assets/images/fever.svg';
+import Diarrhoea from '../../../assets/images/diarrhoea.svg';
+import RunnyNose from '../../../assets/images/runny-nose.svg';
+import ShortnessBreath from '../../../assets/images/shortness-breath.svg';
+
+
+
+const SymptomsCarousel = () => {
+    return (
+        <React.Fragment>
+          <section className='Carousel'>
+            <h4 className='Carousel__title'>Syptoms</h4>
+            <div className='Carousel__container'>
+              <SymptomsCards 
+                image={Tiredness}
+                title={'Tiredeness'}
+                description={'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam'}
+              />
+              <SymptomsCards 
+                 image={Cough}
+                 title={'Cough'}
+                 description={'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam'}
+              />
+              <SymptomsCards
+                  image={Fever}
+                  title={'Fever'}
+                  description={'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam'}
+              />
+              <SymptomsCards
+                  image={Diarrhoea}
+                  title={'Diarrhoea'}
+                  description={'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam'}
+              />
+              <SymptomsCards
+                  image={RunnyNose}
+                  title={'Runny Nose'}
+                  description={'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam'}
+              />
+              <SymptomsCards
+                  image={ShortnessBreath}
+                  title={'Shortness Breath'}
+                  description={'Lorem ipsum laber et sae put ghamma nostra candeat abscurata nun cuam'}
+              />  
```

Comment:
You can reduce this repetition mapping the `symptoms` array
```js
{
  symptoms.map(symtom => <SymptomsCards
     title={symtom.title}
     description={symtom.description}
     image={symtom.image}
     key={symtom.title}/>)
}
```

Reply:
Done! i will be uploading it later today.

Reply:
There are 6 images for the symptoms but in the design it shows only 5 cards, should I make it 6 or 5 cards? Also, is the mobile design going to be a carousel or are we going to show them as one on top of the other?

Reply:
I uploaded the code to feature/cards branch displaying the information using array.map as you recommend and destructured props for a cleaner code. Im waiting for your response on the cards design to finish it. Let me know!

Reply:
I think is better to show them in a carousel, less vertical scroll. We can put the 6 images, adjust them to fit in the layout.

Reply:
0k, will do!